### PR TITLE
com.openai.unity 3.0.0

### LINKED ISSUE #35: no support for new model, "gpt-3.5-turbo"
# Bug Report
When I try to suggest OpenAI's latest model, ""gpt-3.5-turbo", I get an error that the model is not found. Since I am sending a string request to openai, shouldn't support for that not have to be hardcoded? In any case, I can't use the latest model with this package.

## To Reproduce
model = await api.ModelsEndpoint.GetModelDetailsAsync("gpt-3.5-turbo");

Result: HttpRequestException: CreateCompletionAsync Failed! HTTP status code: NotFound.

### PR DESCRIPTION
**Pull request by StephenHodgson**
Closes #35

### CODE REVIEW

File: OpenAI/Packages/com.openai.unity/Documentation~/README.md [hunk 1]
```diff
@@ -105,7 +108,7 @@ You can save the key directly into a scriptable object that is located in the `A
 
 You can create a new one by using the context menu of the project pane and creating a new `OpenAIConfigurationSettings` scriptable object.
 
-![Create new OpenAIConfigurationSettings](images/create-scriptable-object.png)
+![Create new OpenAIConfigurationSettings](OpenAI/Packages/com.openai.unity/Documentation~/images/create-scriptable-object.png)
```
Comment (StephenHodgson) [inline OpenAI/Packages/com.openai.unity/Documentation~/README.md:111]:
```suggestion
![Create new OpenAIConfigurationSettings](images/create-scriptable-object.png)
```

File: OpenAI/Packages/com.openai.unity/Documentation~/README.md [hunk 2]
```diff
@@ -226,6 +229,34 @@ await foreach (var token in api.CompletionsEndpoint.StreamCompletionEnumerableAs
...
+var chatPrompts = new List<ChatPrompt>
+{
+new ChatPrompt("system", "You are a helpful assistant."),
+new ChatPrompt("user", "Who won the world series in 2020?"),
+new ChatPrompt("assistant", "The Los Angeles Dodgers won the World Series in 2020."),
+new ChatPrompt("user", "Where was it played?"),
+};
+var chatRequest = new ChatRequest(chatPrompts);
+var result = await api.ChatEndpoint.GetCompletionAsync(chatRequest);
+Debug.Log(result.Fi
```
Comment (EvanCrabtree) [inline OpenAI/Packages/com.openai.unity/Documentation~/README.md:251]: unfinished line?

File: OpenAI/Packages/com.openai.unity/Documentation~/README.md [hunk 3]
```diff
@@ -226,6 +229,34 @@ await foreach (var token in api.CompletionsEndpoint.StreamCompletionEnumerableAs
...
+```
+
+##### [Chat Streaming](https://platform.openai.com/docs/api-reference/chat/create#chat/create-stream)
+
+```csharp
+TODO
```
Comment (EvanCrabtree) [inline OpenAI/Packages/com.openai.unity/Documentation~/README.md:257]: left todo in

File: OpenAI/Packages/com.openai.unity/Documentation~/README.md (hunk 2 above)
Comment (StephenHodgson) [reply OpenAI/Packages/com.openai.unity/Documentation~/README.md:251]: Yeah I already knocked this out in main. Thanks

File: OpenAI/Packages/com.openai.unity/Documentation~/README.md (hunk 3 above)
Comment (StephenHodgson) [reply OpenAI/Packages/com.openai.unity/Documentation~/README.md:257]: Next version :)

**Pull request merged**